main/src/com/google/refine/util/ParsingUtilities.java

```diff
@@ -229,7 +229,8 @@ static public LocalDateTime stringToLocalDate(String s) {
         if (parsed == null) {
             return null;
         }
-        return parsed.toLocalDateTime();
+        return parsed.withOffsetSameInstant(OffsetDateTime.now().getOffset())
```

Comment:
Is it necessary to return another copy of the offsetDateTime with the same instant value? If so, could you please add a UT to prove the previous way does NOT work?

Reply:
Sure, there you go.

Reply:
:+1: